pkg/exec/local.go

```diff
@@ -36,13 +39,15 @@ var _ Cmder = &LocalCmder{}
 
 // Command returns a new exec.Cmd backed by Cmd
 func (c *LocalCmder) Command(name string, arg ...string) Cmd {
+	klog.V(2).Infof("⚙️ %s %s", name, strings.Join(arg, " "))
 	return &LocalCmd{
 		Cmd: osexec.Command(name, arg...),
 	}
 }
 
 // CommandContext returns a new exec.Cmd with the context, backed by Cmd
 func (c *LocalCmder) CommandContext(ctx context.Context, name string, arg ...string) Cmd {
+	klog.V(2).Infof("⚙️ %s %s", name, strings.Join(arg, " "))
```

Comment:
This might cause issues rendering in some terminals, but we can add handling for `TERM=dumb` later if we run into any issues.